src/lib.rs

```diff
@@ -154,11 +146,27 @@ fn compare_with_latest(
     update_time(date_time, name)
 }
 
-/// Validates current version of the crate
-/// Takes the current name and version
-/// Takes a std::time::Duration to determine the interval
-///         So that it does not notify the user every run
-/// Prints directly to stdout (Will probably change to be more asynchrounos)
+/// Checks if there exists an update by checking against crates.io and notifies the user by printing to stdout
+///
+/// Takes:
```

Comment:
this doesn't format that well it seems via `cargo doc`

![image](https://user-images.githubusercontent.com/128755/82926716-99cf0180-9f4d-11ea-9633-28b543e6b12c.png)

Reply:
Yikes, actually probably just gonna keep that one line and remove the arguments. they should be self explanatory

Reply:
@tarikeshaq try this:

```diff
diff --git a/src/lib.rs b/src/lib.rs
index d036650..8a20790 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -148,13 +148,11 @@ fn compare_with_latest(
 
 /// Checks if there exists an update by checking against crates.io and notifies the user by printing to stdout
 ///
-/// Takes:
+/// # Arguments:
 ///
-///   name: &str representing the name of the crate, you can use env!("CARGO_PKG_NAME")
-///
-///   current_version: &str the version of the CLI, use env!("CARGO_PKG_VERSION")
-///
-///   interval: std::time::Duration representing the interval duration
+///   * `name` - representing the name of the crate, you can use `env!("CARGO_PKG_NAME")`
+///   * `current_version` - the version of the CLI, use `env!("CARGO_PKG_VERSION")`
+///   * `interval` - std::time::Duration representing the interval duration
 ///
 /// # Examples
 ///

```

Reply:
Example here: https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html

Reply:
That makes sense 🤦